content/react/concepts/components/components.md

```diff
@@ -18,7 +18,7 @@ In React, components make up the UI of the application. Components are pieces of
 
```

Comment:
My suggestions:

Typo also found on line # 3. Let's rephrase the `Description` like so:

```
Description: 'Components are pieces of code that are reusable, independent, and make up the UI of a React application.'
```

On line # 14, let's make some edits including emphasizing the first mention of "components" by **bolding** it, like so:

```
In React, **components** are pieces of reusable, independent code that make up the user interface (UI) of the application. Each component comes with one or both of the following:
```